build.sbt

```
@@ -112,6 +113,29 @@ lazy val minitest = crossProject(JVMPlatform, JSPlatform)
 lazy val minitestJVM = minitest.jvm
 lazy val minitestJS = minitest.js
 
+lazy val munit = crossProject(JVMPlatform, JSPlatform)
+  .in(file("munit"))
+  .settings(
+    standardSettings ++ Seq(
+      name := "hedgehog-munit",
+      libraryDependencies ++= Seq("org.scalameta" %%% "munit" % props.MunitVersion) ++
+        (if (scalaBinaryVersion.value.startsWith("2.11")) {
+          val silencerVersion = "1.7.8"
+          Seq(
+            "org.scala-lang.modules" %% "scala-collection-compat" % "2.7.0",
+            compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+            "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full,
+          )
+        } else {
+          Seq.empty
+        })
```

```suggestion
        }),
      Compile / scalacOptions := {
        val options = (Compile / scalacOptions).value
        if (scalaVersion.value.startsWith("3"))
          options.filterNot(_ == "-deprecation")
        else
          options
      }
```
Since Scala 3.1 supports `@nowarn`, I'll bump Scala 3.0.0 to 3.1.x but I'd like to release the version supporting 3.0 before doing that. So let's make it simpler by removing `-deprecation` temporarily for only next release.